### ISSUE TITLE
Marking lacunae

### ISSUE THREAD

**Issue by stenskjaer** [lombardpress/lombardpress-schema#68]
I remember we spent quite a bit of energy in Barcelona returning to the question of `gap`s and `space`s, but I don't know if we ever agreed on how we encode textual *lacunae*. By *lacuna* I here mean that there is no obvious physical holes, spaces or anything in any of the witnesses, but based on the transmitted text, I have the suspicion or conviction as editor that the text is incomplete. This is not the same as the situation where I would mark the text with a *crux* (✝), because the transmitted text is not merely incomprehensible, there is rather something missing.

My immediate idea is to use `gap`, as it indicates a gap in the text or transcription (not on or in the material transcribed, as is the case with `space`), and could imagine something like this:
``` xml
If I have some text where <gap reason="lacuna" />
```
> *lac. statui post* where (or something similar)

There is a possibility of indicating the extent of the lacuna, but you might not know the extent of the *lacuna*.

How does this match your intuitions?

**Comment by jeffreycwitt** [comment]
@stenskjaer I'm unsure where we are with this? 

I can't find any description of our solution in our current docs. 
Should we push this off to 1.1 as a "feature addition"?

I'm assigning this 1.1.0, but change it to 1.0.0 if you think we need to address it immediately.

**Comment by stenskjaer** [comment]
I don't believe we have come to an agreement on this yet. I have started using the suggestion approach (`gap[@type='lacuna']`) in my texts. 

I can introduce it to the documentation if nobody wants to present strong arguments against it.

**Comment by jeffreycwitt** [comment]
It's hard to remember. But I think that sounds fine. 

We do have some guidelines regarding gap already written here: http://lombardpress.org/schema/docs/diplomatic/#unclear-and-illegible-readings

In these cases we've discussed gap as an indicator of unclear text for which the editor does not offers a suggestion. In my mind it should be distinguished from `<space>` which should be reserved for a when a scribe actually leaves a physical blank space to be filled in later.

But I think the `@type=lacuna` will serve to differentiate the above cases of `gap` and your new case where the issues is no longer an unclear text, but a conjecture about missing text. 

If the type is lacuna, there should be no physical dimension attributes. If there is no type then we would expect physical dimensions. This would represent the default type, but we should still have a name for it, to distinguish it from lucana. Perhaps `@type=unclear`. Since in the existing guidelines we suggested the `unclear` tag could be used for a simple case where the editor has a suggestion, and gap when the text is unclear and the editor does not have a suggestion.
